src/Compilers/VisualBasic/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.vb

```diff
@@ -556,6 +556,163 @@ True
 ]]>
 
             Dim verifier As CompilationVerifier = CompileAndVerify(source, expectedOutput)
```

Comment:
Scratch that. The test code for async method was executed more than once, it's just that the implementation of mock runtime is different. I have changed the mock runtime to throw in such scenario as well.